packages/ui-themes/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Instructure, Inc. Engineering and Product Design",
   "type": "module",
   "module": "./es/index.js",
-  "main": "./lib/index.js",
+  "main": "./es/index.js",
```

Comment:
Just wondering why was this changed to the ES module source?

Reply:
There is no `lib/index.js`